tests/JMS/Serializer/Tests/Serializer/xml/array_empty.xml

```diff
@@ -0,0 +1,4 @@
+<?xml version="1.0" encoding="UTF-8"?>
+<result>
+  <entry/>
```

Comment:
Not sure about this. There is no entry after all? If you are not sure, let's skip this test for XML. That's better than adding a potentially wrong test.